webhook-service/lib/template_engine_test.go

```diff
@@ -45,7 +45,7 @@ func TestTemplateEngine_ParseTemplate(t1 *testing.T) {
 			},
 			want:    "",
 			wantErr: true,
-			errMsg:  "unexpected",
+			errMsg:  "template: :1: bad character U+007D '}'",
```

Comment:
why is this change included?

Reply:
It looks like that with go 1.19 the std library to handle templating has better error messages

Reply:
ok cool